docs/installation/windows.md

```diff
@@ -9,122 +9,296 @@ parent = "smn_engine"
 
-This should download the very small `hello-world` image and print a
-`Hello from Docker.` message.
 
 ## Using Docker from Windows Command Line Prompt (cmd.exe)
 
-Launch a Windows Command Line Prompt (cmd.exe).
+1. Launch a Windows Command Line Prompt (cmd.exe).
 
-Boot2Docker command requires `ssh.exe` to be in the PATH, therefore we need to
-include `bin` folder of the Git installation (which has ssh.exe) to the `%PATH%`
-environment variable by running:
+  The `docker-machine` command requires `ssh.exe` in your `PATH` environment
+  variable. This `.exe` is in the MsysGit `bin` folder. 
 
-    set PATH=%PATH%;"c:\Program Files (x86)\Git\bin"
+2. Add this to the `%PATH%` environment variable by running:
 
-and then we can run the `boot2docker start` command to start the Boot2Docker VM.
-(Run `boot2docker init` command if you get an error saying machine does not
-exist.) Then copy the instructions for cmd.exe to set the environment variables
-to your console window and you are ready to run docker commands such as
-`docker ps`:
+        set PATH=%PATH%;"c:\Program Files (x86)\Git\bin"
+        
+2. Create a new Docker VM.
 
-![](/installation/images/windows-boot2docker-cmd.png)
+        docker-machine create --driver virtualbox my-docker-vm
+        Creating VirtualBox VM...
+        Creating SSH key...
+        Starting VirtualBox VM...
+        Starting VM...
+        To see how to connect Docker to this machine, run: docker-machine env my-docker-vm
 
-## Using Docker from PowerShell
+    The command also creates a machine configuration in the
+    `~/.docker/machine/machines` directory. You only need to run the `create`
+    command once. Then, you can use `docker-machine` to start, stop, query, and
+    otherwise manage the VM from the command line.
+    
+2. List your available machines.
 
-Launch a PowerShell window, then add `ssh.exe` to your PATH:
+        docker-machine ls
+        NAME                ACTIVE   DRIVER       STATE     URL                         SWARM
+        my-docker-vm        *        virtualbox   Running   tcp://192.168.99.101:2376  
+        
+    If you have previously installed the deprecated Boot2Docker application or
+    run the Docker CLI, you may have a `dev` VM as well.  When you
+    created `my-docker-vm`, the `docker-machine` command provided instructions
+    for learning how to connect the VM.
 
-    $Env:Path = "${Env:Path};c:\Program Files (x86)\Git\bin"
+3. Get the environment commands for your new VM.
 
-and after running the `boot2docker start` command it will print PowerShell
-commands to set the environment variables to connect to the Docker daemon
-running inside the VM. Run these commands and you are ready to run docker
-commands such as `docker ps`:
+        docker-machine env my-docker-vm
+        export DOCKER_TLS_VERIFY="1"
+        export DOCKER_HOST="tcp://192.168.99.101:2376"
+        export DOCKER_CERT_PATH="/Users/mary/.docker/machine/machines/my-docker-vm"
```

Comment:
Is this path `/Users/....` from a Windows machine as well, or `/c/Users/...`

(I don't know, haven't run it on Windows yet)